.github/workflows/dotnet.yml

```diff
@@ -47,6 +47,11 @@ jobs:
         & dotnet tool update --global uno.check --version $env:UnoCheck_Version --add-source https://api.nuget.org/v3/index.json
         & uno-check -v --ci --non-interactive --fix --skip xcode --skip gtk3 --skip vswin --skip vsmac --manifest $env:UnoCheck_Manifest
 
+    # UWP-Windows project is not supported by `dotnet build`
+    - name: Drop UWP-Windows Project
+      run: |
+        dotnet sln src\ remove src\TestApp\uwp\Uno.UI.RuntimeTests.Engine.UWP
```

Comment:
Let's use solution filters instead. The winui app should not be building either with dotnet build.